src/content/docs/guides/input/1-mouse-button-inputs.mdx

```diff
@@ -1,927 +1,1371 @@
+  <TabItem label="C#">
+
+  | Description             | Enum Value in Code         |
+  | ----------------------- | ---------------------------|
+  | No mouse button         | MouseButton.NoButton       |
+  | The left mouse button   | MouseButton.LeftButton     |
+  | The middle mouse button | MouseButton.MiddleButton   |
+  | The right mouse button  | MouseButton.RightButton    |
+  | The x1 mouse button     | MouseButton.MouseX1Button  |
+  | The x2 mouse button     | MouseButton.MouseX2Button  |
+
+  </TabItem>
+  <TabItem label="Python">
+
+  | Description             | Enum Value in Code         |
+  | ----------------------- | ---------------------------|
+  | No mouse button         | MouseButton.no_button       |
+  | The left mouse button   | MouseButton.left_button     |
+  | The middle mouse button | MouseButton.middle_button   |
+  | The right mouse button  | MouseButton.right_button    |
+  | The x1 mouse button     | MouseButton.mouse_x1_button  |
+  | The x2 mouse button     | MouseButton.mouse_x2_button  |
+
+  </TabItem>
+</Tabs>
+
+### Mouse Clicked
+
+[Mouse Clicked](/api/input/#mouse-clicked) is a boolean function that checks if the specified `Mouse Button` has been clicked since the last time [Process Events](/api/input/#process-events) was called.
+
+Here is a simple example using the left and right mouse buttons:
+
+<Tabs syncKey="code-language">
+  <TabItem label="C++">
+
+  ```cpp
+  #include "splashkit.h"
+
+  int main()
+  {
+      open_window("Mouse Inputs", 800, 600);
+
+      while (!window_close_requested("Mouse Inputs"))
+      {
+          // process user input events
+          process_events();
+
+          // change screen colour to red if left mouse button is clicked
+          if (mouse_clicked(LEFT_BUTTON))
+          {
+              clear_screen(COLOR_RED);
+          }
+          // change screen colour to yellow if right mouse button is clicked
+          else if (mouse_clicked(RIGHT_BUTTON))
+          {
+              clear_screen(COLOR_YELLOW);
+          }
+
+          refresh_screen(60);
+      }
+
+      close_all_windows();
+
+      return 0;
+  }
+  ```
+
+  </TabItem>
+  <TabItem label="C#">
+
+  ```csharp
+  using SplashKitSDK;
+
+  SplashKit.OpenWindow("Mouse Inputs", 800, 600);
+
+  while (!SplashKit.WindowCloseRequested("Mouse Inputs"))
+  {
+      // process user input events
+      SplashKit.ProcessEvents();
+
+      // change screen colour to red if left mouse button is clicked
+      if (SplashKit.MouseClicked(MouseButton.LeftButton))
+      {
+          SplashKit.ClearScreen(SplashKit.ColorRed());
+      }
+      // change screen colour to yellow if right mouse button is clicked
+      else if (SplashKit.MouseClicked(MouseButton.RightButton))
+      {
+          SplashKit.ClearScreen(SplashKit.ColorYellow());
+      }
+
+      SplashKit.RefreshScreen(60);
+  }
+
+  SplashKit.CloseAllWindows();
+  ```
+
+  </TabItem>
+  <TabItem label="python">
+  
+  ```python
+import splashkit as sk

```

Comment:
In the tutorials on guides on Splashkit that have python examples they all us `from splashkit import *` 

It would be best if the python codes in this tutorial follow the same structure as in the other guides, instead of using the `import splashkit as sk`